.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -104,7 +104,6 @@ if [ "$REPO" == "terraform-google-conversion" ]; then
     rm -rf ./tfplan2cai/testdata/templates/
     rm -rf ./tfplan2cai/testdata/generatedconvert/
     rm -rf ./tfplan2cai/converters/google/provider
-    rm -rf ./tfplan2cai/converters/google/resources
```

Comment:
could you leave this line in place? It won't actually fix the problem to revert it

Reply:
since the things that were deleted will stay deleted.

Reply:
and the deletion also isn't the cause of the error

Reply:
OK. How to pass the tgc-base failure?

Reply:
tgc-base will fail, you'll have to merge it anyway.

Reply:
OK. I will close this PR. Thanks.